spec/factories/transactions.rb

```diff
@@ -2,7 +2,7 @@
   factory :transaction do
     invoice { FactoryGirl.create(:invoice) }
     credit_card_number { rand(10 ** 16) }
-    result { ['success', 'failed'].sample }
+    result { "success" }
```

Comment:
@concach 
**Running `rspec` before would _occasionally_ result in a failure or two because _sometimes_ transactions were being created with 'failed' as the result. This should solve that problem. However, there is still a chance that sometimes tests will fail because of how we're setting the `created_at` and/or `updated_at` attribute in our factories.**